packages/hub/test/graphql/commonQueries.ts

```diff
@@ -108,7 +108,7 @@ async function createModel({
   `);
 
   return await executeCommonOperation(mutation, {
-    variables: { input: { groupSlug, slug, code: "2+2" } },
+    variables: { input: { groupSlug, slug, code: "2+2", version: "dev" } },
```

Comment:
All changes in `test/` are unrelated to this PR. I just didn't notice before that tests were broken, because we don't run them in CI yet.